api/nodes/views.py

```diff
@@ -179,13 +241,105 @@ def perform_create(self, serializer):
 
 
 class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
-    """Projects and component details.
+    """Details about a given node (project or component). *Writeable*.
 
     On the front end, nodes are considered 'projects' or 'components'. The difference between a project and a component
-    is that a project is the top-level node, and components are children of the project. There is also a category field
-    that includes the option of project. The categorization essentially determines which icon is displayed by the
-    Node in the front-end UI and helps with search organization. Top-level Nodes may have a category other than
-    project, and children nodes may have a category of project.
+    is that a project is the top-level node, and components are children of the project. There is also a [category
```

Comment:
The problem is that the Node Detail docs are only available when you are on a node detail page which requires a known node_id.  So if I'm just browsing the api, I need to have created a node already in order to view the docs for it.  To avoid that, I decided to copy and paste certain sections and mark them as such with HTML comments.  I'm not crazy about the duplication, but I don't see a way around it with BAPI.